DFC.Composite.Shell.IntegrationTests/Tests/ExternalApplicationTests.cs

```diff
@@ -26,5 +27,21 @@ public async Task CanRedirectToExternalUrl()
             Assert.Equal(HttpStatusCode.Found, response.StatusCode);
             Assert.Equal("http://www.externalpath1.com/", response.Headers.Location.AbsoluteUri);
         }
+
+        [Theory]
+        [InlineData ("product; (product with ;)")]
+        [InlineData("product (product with out ;)")]
+        public async Task CanRedirectToExternalUrlWithUserAgent(string userAgent)
```

Comment:
Can you add a test with inline data containing a trailing ';' to ensure it doesn't fail please

Reply:
Further explanation from gurney, Thsi is ok